elasticdl/proto/elasticdl.proto

```diff
@@ -1,146 +1,162 @@
 syntax = "proto3";
 
 import "google/protobuf/empty.proto";
-import "elasticdl/proto/tensor_dtype.proto";
 
 package master;
 option go_package = "elasticdl/pkg/proto";
 
+enum TensorDtype {
```

Comment:
Probably use a separate PR to combine `tensor_dtype.proto` and `elasticdl.proto` together.

Reply:
Please refer to #1638